i18n/translations/en.json

```diff
@@ -169,7 +169,6 @@
   "error.user.password.notSame": "The passwords do not match",
   "error.user.password.undefined": "The user does not have a password",
   "error.user.role.invalid": "Please enter a valid role",
-  "error.user.undefined": "The user cannot be found",
```

Comment:
I wouldn't remove this. This is also a useful message for the panel and I think we even use it somewhere else.

Reply:
Before I removed it, I searched for `user.undefined` across the codebase and didn't find any remaining match. The error message is a duplicate of `error.user.notFound`.